src/main/webapp/vendor/startbootstrap/index.html

```diff
@@ -252,7 +252,10 @@ <h5 class="modal-title" id="exampleModalLabel">Ready to Leave?</h5>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.20.1/moment.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/chance/1.0.13/chance.min.js"></script>
   <script src="https://uicdn.toast.com/tui-calendar/latest/tui-calendar.js"></script>
-  <script src="script.js"></script>
+
+  <!-- Our scripts -->
+  <script type="module" src="selectScript.js"></script>
```

Comment:
this is cool! Just note that this is a relatively recent feature: https://caniuse.com/#feat=mdn-javascript_statements_import

You probably wouldn't want to depend on this for a real production website just yet, but it works perfectly fine for our use cases here.